App/app/android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
why is this needed??

Reply:
These all are pre generated files from android studio ideally they shouldn't have been pushed and gitignored from beginning.

Reply:
Please remove it :D